app/views/projects/_settings_sidebar.html.erb

```diff
@@ -0,0 +1,24 @@
+<li class="sidebar-item sidebar-brand">
+  PROJECT SETTINGS
```

Comment:
You can use just `Project settings` here and make it uppercase with CSS, I think it's easier to manage, what do you think?